enhancements/update/available-update-metadata.md

```diff
@@ -0,0 +1,266 @@
+This enhancement adds a new `Release` type to be used in `status` properties so we can distinguish between input and output schema.
+
+## Motivation
+
+[Cincinnati][] provides [metadata about releases][cincinnati-metadata], including (since [here][cincinnati-channel-metadata]) the channels to which the release currently belongs.
+Exposing that information to in-cluster consumers allows them to use live metadata, instead of talking to Cincinnati directly or hard-coding expected values.
+For example, the web console currently [hard-codes an expected set of available channels][console-channels] for a given release; with this enhancement they could look up available channels in `status.desired.channels`.
+
+### Goals
+
+* Exposing channel and homepage metadata provided by the upstream update recommendation service for the currently-desired and available-update releases.
+* Documenting the well-known metadata keys for channels and homepages.
+
+### Non-Goals
+
+* Restricting metadata keys to those documented in this enhancement.
+    Update recommendation services such as [Cincinnati][] may continue to add and remove keys as they see fit as long as they do not use different semantics for the keys defined by this enhancement.
+* Propagating metadata keys beyond those documented in this enhancement.
+    As discussed in [the *pass-through metadata maps* section](#pass-through-metadata-maps), capturing additional information from Cincinnati metadata will require additional enhancements and new `Release` properties.
+* Teaching Cincinnati to support requests without `channel` query parameters.
+    This might be useful for recovering from users specifying [`desiredUpdate`][api-desiredUpdate] that were not in their configured [`channel`][api-channel], but I am fine leaving recovery to the users.
+    They should have received sufficient information to adjust their channel from wherever they received the target release they are supplying.
+
+## Proposal
+
+As proposed in [this API pull request][api-pull-request], this enhancement adds a new type:
+
+```go
+// Release represents an OpenShift release image and associated metadata.
+// +k8s:deepcopy-gen=true
+type Release struct {
+  // version is a semantic versioning identifying the update version. When this
+  // field is part of spec, version is optional if image is specified.
+  Version string `json:"version"`
+
+  // image is a container image location that contains the update. When this
+  // field is part of spec, image is optional if version is specified and the
+  // availableUpdates field contains a matching version.
+  Image string `json:"image"`
+
+  // Homepage is the homepage for this release image.  For example,
+  // this could link errata or other release notes describing the release
+  // image.
+	Homepage URL
+
+	// Channels is the set of Cincinnati channels to which the release
+	// currently belongs.
+	Channels []string
```

Comment:
The indentation seems different here, is it intentional?

Reply:
Oops.  Fixed with 120eba5 -> 6633aca.